app/partials/settings/preferences.jade

```diff
@@ -55,7 +55,8 @@ h5.well.em-400.type-h5.hidden-xs(translate="PREFERENCES_EXPLAIN")
       p.text.em-300.hidden-xs(translate="BITCOIN_CURRENCY_EXPLAIN")
     .col-sm-6.col-md-3
     .col-sm-6.col-md-3.setting-result
-      async-select(selected="settings.btcCurrency" range="btcCurrencies" display-prop="code" display-optional="btcValue" on-change="changeBTCCurrency")
+      .list-group.list-group-horizontal
+        a.list-group-item.text-left.brdrn(ng-repeat="currency in btcCurrencies" ng-class="settings.btcCurrency.code === '{{currency.code}}' ? 'active' : ''" ng-click="changeBtcUnit(currency)") {{ currency.code }} ({{ currency.btcValue }})
```

Comment:
Can we just do `ng-click="changeBTCCurrency(currency)"`, then get rid of $scope.changeBtcUnit in preferences.controller.js?